README.dist.md

```diff
@@ -1,46 +1,71 @@
 # WordPress Site Starter
-To use this, clone the WordPress Site Starter into the new project repo. Update this to have info about things to note how to be successful using the starter.
+This is the README for your new site. Feel free to update any of this info to match your project. 
+
+## Links
+-   [Production](#UPDATETHIS)
+-   [Staging](#UPDATETHIS)
+-   [Development](#UPDATETHIS)
 
 ## Requirements
 * [Composer](https://getcomposer.org/) - [Installation](https://getcomposer.org/doc/00-intro.md#installation-linux-unix-macos)
 * [DDEV](https://ddev.readthedocs.io/en/stable/) - [Installation](https://ddev.readthedocs.io/en/stable/users/install/ddev-installation/)
-* [Docker](https://docs.docker.com/desktop/install/mac-install/) (or compatible container alternative)
+* [Docker](https://docs.docker.com/desktop/install/mac-install/) (or compatible container alternative like  OrbStack)
 * For ACF Pro, create an `auth.json` file in `wp-content/mu-plugins/viget-base/` from the [ACF Website](https://www.advancedcustomfields.com/my-account/view-licenses/). (Credentials are in 1Password)
 
+
 ## Setup and Running
 
-Download and install WordPress core files if this is the first time running DDEV
-`ddev wp core download`
+Run `ddev start` to start the site. This will install WordPress files, composer packages, npm packages, and start the DDEV server. Once the server is started DDEV will automatically start Vite for local development. 
+
+If this is the first time this project has been set up: 
+1. Select the desired language.
+2. Fill out the site information.
+3. Then click "Install WordPress"
+4. Once WordPress has been set up login with your user information.
```

Comment:
This step is no longer applicable, however during the `composer create-project` commands, it will prompt for Installation variables like Name, Description (Tagline), and that sort of stuff.

Reply:
Actually, we should make sure this file is worded in a way that a developer is setting up an existing project for the first time. The first dev would need to know that, but after that, WordPress would already be setup and there would be an established Database somewhere (assuming). So Maybe just remove this, but if we did keep it, it would need to be reworded a bit and put in the regular `README.md` file.